activefires_pp/tests/test_spatiotemporal_alarm_filtering.py

```diff
@@ -400,3 +432,26 @@ def test_alarm_filter_runner_call_spatio_temporal_alarm_filtering_no_alarms(crea
     dummy_msg = None
     result = alarm_runner.spatio_temporal_alarm_filtering(dummy_msg)
     assert result is None
+
+
+def test_send_alarm_post_ok(fake_past_detections_dir):
+    """Test send alarm."""
+
+    alarm = json.loads(PAST_ALARMS_MONSTERAS3)
+    restapi_url = "https://httpbin.org/post"
+
+    retv = post_alarm(alarm, restapi_url)
+    assert retv is True
```

Comment:
This might not work when httpbin.org is down. I can recommend mocking requests, for example with the module `responses` https://pypi.org/project/responses/

Reply:
Ahh, nice, thanks, will have a look